CA-200934: Fix network boot with GVT-g

ROMBIOS has a check disallowing any PCI CONFIG calls on hardware without
an i440FX host bridge.  With GVT-g, the host bridge in the VM is set to
match the actual hardware which then prevents iPXE from attempting to
PXE boot.  See vgt_bridge_pci_conf_init() in QEMU for where this is
done.

To fix it, disable the check in ROMBIOS.

diff --git a/tools/firmware/rombios/rombios.c b/tools/firmware/rombios/rombios.c
index 74ec8f2cda99..b8ef761e54ae 100644
--- a/tools/firmware/rombios/rombios.c
+++ b/tools/firmware/rombios/rombios.c
@@ -219,7 +219,7 @@
 // define this if you want to make PCIBIOS working on a specific bridge only
 // undef enables PCIBIOS when at least one PCI device is found
 // i440FX is emulated by Bochs and QEMU
-#define PCI_FIXED_HOST_BRIDGE 0x12378086 ;; i440FX PCI bridge
+//#define PCI_FIXED_HOST_BRIDGE 0x12378086 ;; i440FX PCI bridge
 
 // #20  is dec 20
 // #$20 is hex 20 = 32
